map/src/main/java/openlr/map/utils/GeometryUtils.java

```diff
@@ -702,6 +716,21 @@ public static GeoCoordinates determineCoordinateInDistance(
         return new GeoCoordinatesImpl(lon2, lat2);
     }
 
+    /**
+     * Degree of interval between two angles
+     * @param firstBearing angle of line to the north
+     * @param secondBearing angle of line to the north
+     * @return Absolute rounded interval value

```

Comment:
I don't think the return result has the correct description.  Maybe "the absolute difference between the 2 angles".

Reply:
458ee63d9b353f59b5145a2bb129eef2fe2ab6ad